app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -1,6 +1,10 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < ManageIQ::Providers::EmbeddedAutomationManager::Authentication
   FRIENDLY_NAME = "Embedded Ansible Credential".freeze
 
+  def self.credential_type
+    "embedded_ansible_credential_types"
+  end
+
```

Comment:
NOTE I think once we get away from separate "embedded_workflows" and "embedded_ansible" controllers we can drop these "groups" of credential types, because you would just call OPTIONS and pass in a type like we do with ext_management_systems.  So you would as for the credential types for the Workflows AutomationManager or the EmbeddedAnsible AutomationManager.

But for now this keeps compatibility without having to hard code all of the types :+1:

Reply:
I don't thin OPTIONS supports the collection_class, but it _could_.

Reply:
@Fryguy we pass in `type=class_name` to `OPTIONS`, that is how we get DDF params for a particular EMS once you've selected it in the dropdown

Reply:
Sorry, clarification - type is fine for the purpose listed which is an EMS is selected and then we filter on that EMS.  collection_class would be for other purposes.